src/AzureAuth/CommandMain.cs

```diff
@@ -475,6 +475,7 @@ private int GetToken()
 
                 var tokenResult = successfulResult.TokenResult;
                 this.eventData.Add("silent", tokenResult.IsSilent);
+                this.eventData.Add("sid", tokenResult.Sid);

```

Comment:
How do we handle when sid is null?

Reply:
If it is null, it will be blank in telemetry, like `option_resource`. You can compare it with an empty string.

Reply:
Have we ever observed `Sid` to actually _be_ `null`? Is that a possible value? 🤔

Reply:
Lol. Nevermind, I think I got my answer below. 😅

Reply:
SID was used in SourceDepot (according to Kyle). If it is null, SD won't work. 
So, I guess it cannot be null.